### DISCUSSION TITLE
how to make it so weapons shoot grenades?

### DISCUSSION THREAD

**Discussion by gamer3106** [Pinsplash/halflife2chaos#72] [General]
i would like to know if there was a way to set the mod so that the weapons shoot grenades and for that to be the only active effect, like in pinplash's video. 

thanks in advance

**Comment by acuifex** [comment]
i think it's `chaos_replace_bullets_with_grenades 1` and `chaos 0`

**Reply by gamer3106** [reply]
i'll try it out, thx
